internal/api/attestationconfigapi/fetcher.go

```diff
@@ -43,7 +43,13 @@ func NewFetcher() Fetcher {
 
 // NewFetcherWithClient returns a new fetcher with custom http client.
 func NewFetcherWithClient(client apifetcher.HTTPClient) Fetcher {
-	return newFetcherWithClientAndVerifier(client, sigstore.CosignVerifier{})
+	verifier := sigstore.CosignVerifier{}
+	err := verifier.SetPublicKey([]byte(cosignPublicKey))
+	if err != nil {
+		// This relies on an embedded public key. If this key can not be validated, there is no way to recover from this.
+		panic(fmt.Errorf("creating cosign verifier: %w", err))
```

Comment:
Library code should not panic. It seems irrelevant in our current use case, but as a library you can rarley know if a panic is approrpiate (there are some edge cases: https://stonecode.ca/idiomatic-panics/ ). Imagine this running concurrently with other code and panicing would likely stop it from terminating gracefully.